### DISCUSSION TITLE
sveltekit losing context within chrome devtools

### DISCUSSION THREAD

**Discussion by weepy** [sveltejs/kit#6187] [General]
I find that often Chrome devtools I get some issues occasionally

1. The Elements tree becomes blank. Refresh does not work. Only fix : create a new tab. 
2. Sveltekit.svelte source mapping is old. Fix: restart vite or sometimes it's necessary to delete .sveltekit directory. 

This is with latest, but it's been around for a while. 

Has anyone else experienced ?

**Comment by mskocik** [comment]
I experienced this sometimes in Firefox (not using chrome) and not with sveltekit, but with JS library which replaced DOM on navigation requests. Closing and opening dev tools works always worked for me.

So I would say it's probably not directly related to sveltekit 🤔

**Reply by weepy** [reply]
Second one is probably something to do with HMR.
